packages/strapi-plugin-content-type-builder/admin/src/translations/en.json

```diff
@@ -43,6 +43,7 @@
   "error.attribute.key.taken": "This value already exists",
   "error.attribute.sameKeyAndName": "Can't be equal",
   "error.validation.minSupMax": "Can't be superior",
+  "error.attribute.forbidden": "This attribute can't be used",
```

Comment:
This attribute name is reserved? I think it's more understandable and explain why it can't be used